docs/development.md

```diff
@@ -108,6 +108,13 @@ UUIDs may be required in the following fields of a Test object: In, Out, and Con
 
 Create a new [release checklist](https://github.com/coreos/ignition/issues/new?labels=kind/release&template=release-checklist.md) and follow the steps there.
 
+## The build process
+
+Note that the `build` script included in this repository is a convenience script only and not used for the actual release binaries. Those are built using a `ignition.spec` maintained in [Fedora rpms/ignition](https://src.fedoraproject.org/rpms/ignition).
```

Comment:
The published ignition-validate container isn't built from the specfile; the Dockerfile uses `./build_for_container`.

nit: "an `ignition.spec`"

Reply:
Should be fixed now